sources/packages/backend/apps/db-migrations/src/sql/SFASIndividuals/Add-ppd-status-ppd-status-date.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE
+    sims.sfas_individuals
+ADD
+    COLUMN ppd_status BOOLEAN;
+
+COMMENT ON COLUMN sims.sfas_individuals.ppd_status IS 'Persistent or Prolonged Disability Flag (individual.ppd_flg).';
+
+ALTER TABLE
+    sims.sfas_individuals
+ADD
+    COLUMN ppd_status_date DATE;
```

Comment:
Is this a ppd effective date? if so is it appropriate to map it to sims effective date??

Reply:
As column comment it is the PPD effective date. What do you mean by `sims effective date`? Do you think its name should be `ppd_effective_date`?

Reply:
There is a column in students table `disability_status_effective_date` and seeing the story description I thought so.

![image](https://github.com/bcgov/SIMS/assets/54600590/52dfe22f-b067-4050-ae3e-ca50bae84680)

I am ok with column name. My question was is that intended to be mapped to students table to column `disability_status_effective_date`